src/main/java/org/openelisglobal/odoo/config/TestProductMapping.java

```diff
@@ -0,0 +1,143 @@
+package org.openelisglobal.odoo.config;
+
+import jakarta.annotation.PostConstruct;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import lombok.Getter;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+import org.openelisglobal.common.log.LogEvent;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TestProductMapping {
+
+    private static final String ENV_MAPPING_FILE = "ODOO_MAPPING_FILE";
+    private static final String DEFAULT_CSV_FILE = "odoo-test-product-mapping.csv";
```

Comment:
please we dont have to put a config file on the classpath.
Just load this from a fixed external file path
ie. ` /var/lib/openelis-global/odoo/odoo-test-product-mapping.csv`
and a user can always mount a file to that path in the container , in the docker compose file
`- ./volume/odoo/odoo-test-product-mapping.csv:/var/lib/openelis-global/odoo/odoo-test-product-mapping.csv,`